molecule/aws/side_effect.yml

```diff
@@ -9,6 +9,14 @@
     # WHEN REINSTATING REBOOT
     - include: reboot_and_wait.yml
       when: "false"
+    - include: tor_apt_test.yml
+      when: (lookup('env','CIRCLE_BRANCH')|default('na')).startswith('release')
```

Comment:
@msheiny Assigning defaults to env lookups often leads to surprises. I believe it's currently not possible to override an undefined env var with `|default`, but the lookup will return an empty string, which will satisfies the conditional logic here.

Reply:
ohhh shit you are totally right @conorsch - thanks for pointing that out

Reply:
Older versions of Ansible threw an uncatchable exception in the event of an undefined env var lookup, so the behavior has improved somewhat, but the silent ignore of `|default` warrants careful attention. Still, good to go here. 👌